conan/tools/cmake/toolchain/blocks.py

```diff
@@ -518,12 +553,15 @@ def context(self):
             find_package_prefer_config = "OFF"
 
         is_apple_ = is_apple_os(self._conanfile)
+        is_win = self._conanfile.settings.get_safe("os") == "Windows"
+        build_type = self._conanfile.settings.get_safe("build_type")
```

Comment:
``build_type`` seems unused in this method? A linter in your IDE/Editor should most likely highlight this ``build_type`` variable as unused to warn you.